discovery-infra/test_infra/consts.py

```diff
@@ -94,7 +94,8 @@ class Events:
     REGISTERED_CLUSTER = "Registered cluster"
     SUCCESSFULLY_REGISTERED_CLUSTER = "Successfully registered cluster"
     PENDING_FOR_INPUT = "to pending-for-input"
-    GENERATED_IMAGE = "Generated image (SSH public key is set)"
```

Comment:
missed that
Is the message is going to totally replaced? 
cause this will break the current regression run. 
So until this feature get to staging lets leave it, with a remark?

Reply:
Sure, so simply left both consts. When ready, we could just remove the stale one.